src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/UrlIndexFeatureSource.java

```diff
@@ -24,7 +24,6 @@
 import org.joda.time.DateTimeZone;
 import org.joda.time.format.DateTimeFormat;
 import org.joda.time.format.DateTimeFormatter;
-import org.joda.time.format.ISODateTimeFormat;
```

Comment:
Lean't towards not removing this because of the comments it might draw, but did it anyway - its not used after all